python/pyhail/dataset.py

```diff
@@ -435,15 +656,70 @@ def annotate_variants_table(self, path, variant_expr, root=None, code=None, conf
         return self.hc.run_command(self, pargs)
 
     def annotate_variants_vds(self, other, code=None, root=None):
-        """Annotate variants with variant annotations from .vds file.
+        '''Annotate variants with variant annotations from .vds file.
+
+        **Examples**
+
+        The following expression copies the ``anno1`` annotation from ``other``
```

Comment:
I don't think you need "The following expression". Just say "Copy the ``anno1`` annotation from ``other``